components/suggestion/switch_channel_provider.jsx

```diff
@@ -150,7 +159,15 @@ class SwitchChannelSuggestion extends Suggestion {
                 {...Suggestion.baseProps}
             >
                 {icon}
-                <span>{displayName}</span>
+                <span>
+                    {fullname.length > 1 ? fullname : username}
```

Comment:
Hmm this might lead to confusion, how about we rework it be something like:

```
let mainText = username
if (user.firstname) {
	mainText = `${user.last_name} ${user.first_name}`;
}

let secondaryText = ''
if (user.firstname) {
	secondaryText = username;
}


...
```

Thoughts?